src/OctoshiftCLI.Tests/ado2gh/Commands/GenerateScriptCommandTests.cs

```diff
@@ -269,7 +269,7 @@ public async Task SequentialScript_Skips_Team_Project_With_No_Repos()
 
             // Assert
             _scriptOutput.Should().BeNull();
-            _mockOctoLogger.Verify(m => m.LogError("A migration script could not be generated because no migratable repos were found. Please note that the GEI does not migrate disabled and TFVC repos."));
```

Comment:
you can leave it like this if you want, but if I was writing this test I'd just assert that something was written to the error log, not the specific message.  In general I'm not a huge fan of testing specific log output (I know you do that often, but I don't)

Reply:
I am always kinda torn, my philosophy is that the error message is part of the user experience so I add a verification to make sure that if the message changes at least a test breaks. But I also agree that it is not part of the actual functionality of a unit and does not affect the correctness of it.

Reply:
I also believe that some things are more valuable to test than others, and sometimes the cost of writing and maintaining a test isn't justified by the value the test provides.

Reply:
addressed in f56608976fee6799b938eb6d58e587ac66ce6d9f